person.rb

```diff
@@ -0,0 +1,23 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(age, name: 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  private
+
+  def of_age?
+    age >= 18
+  end
+
+  public
+
+  def can_use_services?
+    age >= 18 || @parent_permission
+  end
```

Comment:
- [ ] Since you have created a module of of_age?, I would implore you to just call that method rather than writing this `age >= 18`.